app/views/components/PageCYASubmit.scala.html

```diff
@@ -16,10 +16,11 @@
 
 @import models.common._
 @import templates.helpers.Heading
+@import views.html.components._
 
```

Comment:
All are wildcard imports because I will be making other PRs replacing view components which I will also be importing from `views.html.compontents`